lib/routes/eventbrite/events.ts

```diff
@@ -0,0 +1,152 @@
+import type { Data, DataItem, Route } from '@/types';
+import { toTitleCase } from '@/utils/common-utils';
+import ofetch from '@/utils/ofetch';
+import { parseDate } from '@/utils/parse-date';
+import { load } from 'cheerio';
+import type { Context } from 'hono';
+
+export const route: Route = {
+    path: '/:region/:eventType?/:includePromoted?',
+    categories: ['other'],
+    example: '/eventbrite/canada--toronto/all-events',
+    parameters: { eventType: 'category of events for filtering', region: 'Region or scope of events' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['eventbrite.com/d/:region/:eventType', 'eventbrite.ca/d/:region/:eventType'],
+            target: '/:region/:eventType',
+        },
+    ],
+    name: 'Events',
+    maintainers: ['elibroftw'],
+    handler: async (ctx: Context): Promise<Data> => {
+        const params = ctx.req.param();
+        const { region, includePromoted = 'false' } = params;
+        let eventType = params.eventType || 'all-events';
+        if (eventType === 'events') {
+            eventType = 'all-events';
+        }
+        const link = `https://www.eventbrite.com/d/${region}/${eventType}/`;
+        const response = await ofetch(link, {
+            headers: {
+                'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:138.0) Gecko/20100101 Firefox/138.0',
+            },
```

Comment:
Does the site only work on this fixed version of Firefox on Windows? If the answer is no, please remove it which RSSHub will use a randomised Chrome on macOS by default.